_studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_lin.cpp

```diff
@@ -170,16 +171,19 @@ void DDI_VA::InitAlloc(const FeatureBlocks& /*blocks*/, TPushIA Push)
     Push(BLK_CreateService
         , [this](StorageRW& strg, StorageRW& local) -> mfxStatus
     {
-        const auto&             par     = Glob::VideoParam::Get(strg);
+        auto& core = Glob::VideoCore::Get(strg);
+        const auto& par = Glob::VideoParam::Get(strg);
         const mfxExtHEVCParam&  hevcPar = ExtBuffer::Get(par);
         mfxStatus sts;
 
         m_callVa = Glob::DDI_Execute::Get(strg);
 
-        std::vector<VAConfigAttrib> attrib(2);
+        std::vector<VAConfigAttrib> attrib(3);
+        mfxPriority priority;
```

Comment:
minor: it is actually a codeline waste, better to declare variable where you actually need it, i.e. on line 198
```
mfxPriority priority = core.GetSession()->m_priority;
```

Reply:
That's not actually a minor. This will remove the concern of variable not being initialized. Please, move declaration below.

Reply:
Done.